sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -1,17 +1,11 @@
 # Release History
 
-## 4.8.0-beta.1 (Unreleased)
+## 4.8.0-beta.1 (2023-11-08)
 
 ### Features Added
 
 - Added `hsmPlatform` property to `KeyProperties`.
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-
 ## 4.7.2 (2023-08-10)
 
 ### Bugs Fixed
```

Comment:
Should we mention the default service version here as well just like we do in the other KV packages?

Reply:
oops, yes we should!